InternTrack.Core.Api/Services/Foundations/Interns/InternService.cs

```diff
@@ -5,6 +5,7 @@
 
 using System;
 using System.Linq;
+using System.Net.Sockets;
```

Comment:
I am not sure why it is being used. I hope this is not incorrect using statement.